spec/commands/install_spec.rb

```diff
@@ -498,9 +498,15 @@
       expect(out).not_to include("You're running Bundler 999 but this project uses #{Bundler::VERSION}.")
     end
 
+    it "should print warning if versions do not match" do
+      bundle :init
+      bundle :install, :env => { "BUNDLE_POSTIT_TRAMPOLINING_VERSION" => "999" }
+      expect(out).to include("You're running Bundler 999 but this project uses #{Bundler::VERSION}.")
+    end
+
     it "should not print warning if versions match" do
       bundle :init
-      bundle :install, :env => { "BUNDLE_POSTIT_TRAMPOLINING_VERSION" => "#{Bundler::VERSION}" }
+      bundle :install, :env => { "BUNDLE_POSTIT_TRAMPOLINING_VERSION" => Bundler::VERSION }
       expect(out).not_to include("You're running Bundler #{Bundler::VERSION} but this project uses #{Bundler::VERSION}.")
```

Comment:
this would pass if we added, for example, a newline to the message, so we really want a positive assertion for what's in the output (i.e. is it empty, or what actually gets printed)